README.md

```diff
@@ -35,22 +35,21 @@ Then perform the following commands on the config folder:
 - `terraform apply` to apply the infrastructure build
 - `terraform destroy` to destroy the built infrastructure
 
-[^]: (autogen_docs_start)
-
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| admin\_disable\_polling | Whether to disable polling for Admin API | string | `"false"` | no |
+| admin\_disable\_polling | Whether to disable polling for Admin API | bool | `"false"` | no |
 | admin\_max\_calls | Maximum calls that can be made to Admin API | string | `"14"` | no |
 | admin\_period | The period of max calls for the Admin API (in seconds) | string | `"1.0"` | no |
-| appengine\_disable\_polling | Whether to disable polling for App Engine API | string | `"false"` | no |
+| appengine\_disable\_polling | Whether to disable polling for App Engine API | bool | `"false"` | no |
```

Comment:
Shouldn't the quotes be removed as well around the default value if they're bool?

Reply:
The inputs table and outputs table are generated using [terraform-docs](https://github.com/segmentio/terraform-docs) and a [Terraform 0.12 compatibility wrapper](https://github.com/antonbabenko/pre-commit-terraform/blob/master/terraform_docs.sh). This may be a bug in the latter, but we don't manually edit the content.